### PR TITLE
fetch config and move feedback widget for the plugin

### PR DESCRIPTION
**Pull request by leangseu-edx**
- follow the contract to get waffle flag and available languages from the api
- move feedback widget
- update tests

ticket: https://2u-internal.atlassian.net/browse/AU-1915

### CODE REVIEW

**Review by nsprenkle — CHANGES_REQUESTED** [review]
The data values don't match contract. Please update so they match the contract or, if it makes more sense to change the contract, let's do that, but they must match

**Review by Rodra — APPROVED** [review]
Looks great !

File: plugins/UnitTranslationPlugin/translation-selection/index.jsx [hunk 1]
```diff
@@ -63,18 +71,28 @@ const TranslationSelection = ({
         courseId={courseId}
         selectedLanguage={selectedLanguage}
         setSelectedLanguage={setSelectedLanguage}
+        availableLanguages={availableLanguages}
         id={id}
       />
+      <FeedbackWidget
```
Comment (Rodra) [inline plugins/UnitTranslationPlugin/translation-selection/index.jsx:77]: Why are we rendering the widget here ?

Comment (leangseu-edx) [reply plugins/UnitTranslationPlugin/translation-selection/index.jsx:77]: This is just a place holder. It create the virtual dom here. But it get moved with ref.

Comment (Rodra) [reply plugins/UnitTranslationPlugin/translation-selection/index.jsx:77]: Ohh I see... COOL !

**Review by Rodra — APPROVED** [review]

**Review by nsprenkle — APPROVED** [review]
LGTM but before you merge, can you update PR title? This is no longer just language config, it is also feedback, etc.

**Pull request merged**